### PR TITLE
Add Css

### PR DESCRIPTION
**Pull request by mace-akkari**
Added basic Css to the site

### CODE REVIEW

File: portfolio.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
+body{
+  background-color: lightgray;
+  margin: 0;
+}
+
+.wrapper {
+  max-width: 620px;
+  display: grid;
+  grid-template-columns: repeat (2, 1fr);
+  grid-template-rows: repeat (5, clearçç1fr);
```
Comment (paddyo) [inline portfolio.css]: Looks some bad css got in there!

File: portfolio.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  font-family: 'Graduate' , sans-serif;
+  font-size: 80px;
+}
+
+h2 {
+  font-family: 'Six+Caps' , cursive;
+}
+
+
+footer{
```
Comment (paddyo) [inline portfolio.css:45]: Have you thought about using flexbox to sort the alignment issues?

File: portfolio.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
+body{
+  background-color: lightgray;
+  margin: 0;
+}
+
+.wrapper {
+  max-width: 620px;
+  display: grid;
+  grid-template-columns: repeat (2, 1fr);
+  grid-template-rows: repeat (5, 1fr);
```
Comment (paddyo) [inline portfolio.css]: For this design you can get rid of this and use auto rows.

File: portfolio.css [hunk 4]
```diff
@@ -0,0 +1,53 @@
+body{
+  background-color: lightgray;
+  margin: 0;
+}
+
+.wrapper {
+  max-width: 620px;
+  display: grid;
+  grid-template-columns: repeat (2, 1fr);
```
Comment (paddyo) [inline portfolio.css]: Maybe use 6 columns for now

File: portfolio.css (hunk 2 above)
Comment (mace-akkari) [reply portfolio.css:45]: will do it on a different branch.

**Review by paddyo — APPROVED** [review]

**Comment by paddyo** [comment]
👍

**Pull request merged**